draft-ietf-quic-qpack.md

```diff
@@ -1411,7 +1411,11 @@ weaknesses.
 An implementation has to set a limit for the values it accepts for integers, as
 well as for the encoded length; see {{prefixed-integers}}. In the same way, it
 has to set a limit to the length it accepts for string literals; see
-{{string-literals}}.
+{{string-literals}}.  These limits SHOULD be large enough to process the
+largest individual field the HTTP implementation can be configured to accept.
+
+If an implementation encounters a value larger than it is able to decode,
+this MUST be treated as a stream error of QPACK_DECOMPRESSION_FAILED.
```

Comment:
Would this be a connection error if the too-large literal occurred on the encoder stream?

Reply:
Technically, it could be any of the three error types we define, depending where it occurs.